test/tester.go

```diff
@@ -278,3 +283,20 @@ func (t *Tester) UnpinPOST(sl string) (int, error) {
 	r, err := t.Request(http.MethodPost, "/unpin", nil, body, nil, nil)
 	return r.StatusCode, err
 }
+
+// SweepPOST kicks off a background process which gets all files pinned by skyd
+// and marks them in the DB as pinned by the current server. It also goes over
+// all files in the DB that are marked as pinned by the local skyd and unmarks
+// those which are not in the list reported by skyd.
+func (t *Tester) SweepPOST() (api.SweepPOSTResponse, int, error) {
+	var resp api.SweepPOSTResponse
```

Comment:
For super short methods like these we sometimes use named return arguments. This is fine too though. Just a fun fact.